src/ControlPanel.php

```diff
@@ -221,57 +206,6 @@ public function check_cache(): void
         wp_die();
     }
 
-    /**
-     * Adds a check cache button to check the ENS API.
-     */
-    public function check_engaging_networks(): void
```

Comment:
Should we also remove line 35?
`add_action('wp_ajax_check_engaging_networks', [ $this, 'check_engaging_networks' ]);`